invenio_rdm_records/services/facets.py

```diff
@@ -39,6 +39,13 @@
 )
 
 
+filetype = TermsFacet(
+    field="files.types",
+    label=_("File type"),
+    value_labels=lambda ids: {id: id.upper() for id in ids},
```

Comment:
This might not be the most ideal way of labelling the facet, however I wasn't able to find a potential list of file formats. We can split this out as a separate task to review all the file formats on Zenodo, and check which ones should not be all capital and only document those here.